### PR TITLE
Add draft of 2022-09-08 leadership meeting minutes

### PR DESCRIPTION
**Pull request by jcoliver**
Minutes from 2022-09-08 Trainers Leadership Committee meeting.

### CODE REVIEW

File: leadership_minutes/2022-09-08-leadership-minutes.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+3. Review Trainers Leadership Committee expectations
+    - Karen: will update on-boarding materials to say that proposals are how we 
+    put items on our agenda, can be just an idea.
+    - Under [Trainers Leadership Committee meeting rules](https://github.com/carpentries/trainers/blob/main/policy/leader_meeting_rules.md), "consideration for removal [from the committee] occurs after 3 unexplained 
+    no-shows".
+    - The [Leadership Committee rules](https://github.com/carpentries/trainers/blob/main/governance.md#removal) 
+    state "If a Member consistently fails to meet the obligations outlined 
+    above, they may be removed by a unanimous vote of the other Members". 
+4. Low feedback scores from an event: thinking about follow-up
```
Comment (marklcrowe) [inline leadership_minutes/2022-09-08-leadership-minutes.md:71]: Hi @jcoliver, this item (low feedback scores) was added to the agenda in the wrong place - it's supposed to be one of the core team update items. Please can you move it to that section and update numbering accordingly? Sorry for not editing that on the agenda notes at the time.

**Review by marklcrowe — CHANGES_REQUESTED** [review]
So according to GitHub, my comment below (line 71-73, item 4) isn't actually a comment, and I need to leave a comment requesting changes. So copy-paste as below:

Hi @jcoliver, this item (low feedback scores) was added to the agenda in the wrong place - it's supposed to be one of the core team update items. Please can you move it to that section and update numbering accordingly? Sorry for not editing that on the agenda notes at the time.

**Review by paocorrales — APPROVED** [review]

**Review by bonnyad — APPROVED** [review]

**Review by bonnyad — DISMISSED** [review]

**Review by marklcrowe — DISMISSED** [review]
Thanks for making that correction Jeff

File: leadership_minutes/2022-09-08-leadership-minutes.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+Absent: 
+
+1. Core Team Updates
+    - Checkout project
+        - Need to approve meeting topic :)
+        - Plan to record meeting, make it clear that we are using it only for 
+        assessment record, think we should not make it available to community.
+        - Mark: what will the recording be used for
+        - Karen: for Kelly's reference only
+        - Sarah, and Pao agree to not recording
```
Comment (brownsarahm) [inline leadership_minutes/2022-09-08-leadership-minutes.md:17]:
```suggestion
        - Sarah, and Pao agree to not sharing recording
```

Comment (brownsarahm) [reply leadership_minutes/2022-09-08-leadership-minutes.md:17]: @paocorrales is that what you remember agreeing to? If so, can you commit that? then I'll also approve the minutes

Comment (paocorrales) [reply leadership_minutes/2022-09-08-leadership-minutes.md:17]: Yes, I meant not sharing the recording.

**Review by paocorrales — APPROVED** [review]

**Review by brownsarahm — APPROVED** [review]

**Pull request merged**